src/Symfony/Component/Utf8/Bytes.php

```diff
@@ -0,0 +1,505 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Utf8;
+
+use Symfony\Component\Utf8\Exception\InvalidArgumentException;
+
+final class Bytes implements GenericStringInterface, \Countable
+{
+    private $string = '';
+
+    public static function fromString(string $string): self
+    {
+        $bytes = new self();
+        $bytes->string = $string;
+
+        return $bytes;
+    }
+
+    public static function fromCharCode(int ...$codes): self
```

Comment:
`fromCharCode` is the usual for this function un other programming language.
I guess this is because the usual use case is to give it a single char.
About the no-char case, that will return the empty string.
This should be supported to deal with calls like `fromCharCode(...$a)` where $a ends up being an empty array, which is a valid use case.